electron/main.ts

```diff
@@ -291,16 +292,16 @@ ipcMain.on('getLog', (event, appName) =>
 
 ipcMain.on('removeFolder', async (e, [path, folderName]) => {
   if (path === 'default') {
-    const defaultInstallPath = (await GlobalConfig.get().getSettings()).defaultInstallPath.replaceAll("'", '')
+    const defaultInstallPath = (await GlobalConfig.get()).config.defaultInstallPath.replaceAll("'", '')
     const folderToDelete = `${defaultInstallPath}/${folderName}`
     return setTimeout(() => {
-      exec(`rm -Rf ${folderToDelete}`)
+      rmdirSync(folderToDelete, {recursive: true})
```

Comment:
Have you tested this approach? It was my first try and it didn't work, but maybe I missed the recursive option, I don't remember now.

Reply:
According to the spec, it should behave exactly as `rm -R`. So I think we're good here.

Reply:
Nope, I tested, it doesn't work. Nice catch.

Reply:
Great. `exec` doesn't work as well. I think there's something wrong with my setup. Anyway, I trust the spec, and we can fix it before release if we find out the root cause.

Reply:
have you tried to check if the folder to delete is coming correctly?
Because the exec should work since it worked before.

Reply:
It isn't, I'm investigating.

Reply:
Fixed as discussed on Discord.